changelog/1664.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added the `~plasmapy.formulary.frequencies.plasma_critical_density`
+function to the `~plasmapy.formulary.frequencies` module. This function calculates the critical density of the plasma for a given plasma frequency.
```

Comment:
The derivation uses the plasma frequency, but I think it's more accurate to say "calculates the critical density of a plasma for a given frequency of radiation" or something similar.